roles/tas_single_node/README.md

```diff
@@ -20,6 +20,9 @@ Deploy the [RHTAS](https://docs.redhat.com/en/documentation/red_hat_trusted_arti
 | tas_single_node_podman_network | Name of the Podman network for containers to use. | str |  `rhtas`  |
 | tas_single_node_rekor_redis | Details on the Redis connection for Rekor. You can set this to a custom Redis instance. | dict of 'tas_single_node_rekor_redis' options |  `{'database_deploy': True, 'redis': {'host': 'rekor-redis-pod', 'port': 6379, 'password': 'password'}}`  |
 | tas_single_node_trillian | Details on the database connection for Trillian. You can set this to a custom MySQL or MariaDB instance. | dict of 'tas_single_node_trillian' options |  `{'database_deploy': True, 'mysql': {'user': 'mysql', 'root_password': 'rootpassword', 'password': 'password', 'database': 'trillian', 'host': 'trillian-mysql-pod', 'port': 3306}}`  |
+| tas_single_node_fulcio | Details on the certificate settings for Fulcio. You can configure the certificate with the organization's details. | dict of 'tas_single_node_fulcio' options |  `{'certificate': {'organization_name': '', 'organization_email': '', 'common_name': ''}}`  |
+| tas_single_node_fulcio_private_key | Private key for Fulcio, used for signing root certificate. | str |  |
+| tas_single_node_fulcio_root_ca | The root certificate for Fulcio. | str |  |
```

Comment:
Should this be mutually exclusive with `tas_single_node_fulcio`? If so, then let's mention it here and also create a task that would validate that only one of these is provided.

Reply:
Also, maybe we should also nest `tas_single_node_fulcio_private_key` and `tas_single_node_fulcio_root_ca` under `tas_single_node_fulcio` (on the same level as the `certificate` attribute). I think it would make more sense to set these values at a single place.